coriolis/db/sqlalchemy/migrate_repo/versions/017_adds_user_scripts_to_migration.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+import sqlalchemy
+
+
+def upgrade(migrate_engine):
+    meta = sqlalchemy.MetaData()
+    meta.bind = migrate_engine
+
+    # add 'user_scripts' column to 'migration':
+    migration = sqlalchemy.Table(
+        'migration', meta, autoload=True)
```

Comment:
Please add the column to the base_transfer_action table so we can have Replicas “remember” their user script settings too so users won’t need to specify them each Replica deployment.

Reply:
It is possible to add this to replicas too, but only after a first deployment has been done. Currently, there's no way to add user_scripts when creating a replica, is this what is actually needed here?